modules/farm_fd2_school/src/entrypoints/Vue1/App.vue

```diff
@@ -105,10 +73,17 @@
       size="7"
       class="label-margin"
     />
-    <select id="harvest-units">
-      <option selected>BUNCH</option>
-      <option>EACH</option>
-      <option>POUND</option>
+    <select
+      id="harvest-units"
+      v-model="selectedUnit"
+    >
+      <option
```

Comment:
Looks like you are rendering the crops into the Units select?  Copy and paste error?